Ghidra/Features/Base/src/main/java/ghidra/app/plugin/core/bookmark/DeleteBookmarkAction.java

```diff
@@ -37,7 +37,6 @@
 	/**
 	 * Creates a new action with the given name and associated to the given
 	 * plugin.
-	 * @param id the name for this action.
 	 * @param plugin the plugin this action is associated with.
 	 */
 	DeleteBookmarkAction(BookmarkPlugin plugin, Bookmark bookmark, boolean isOffcut) {
```

Comment:
Well the purpose of the PR was more to not-have parameters in the javadoc which do not exist in the method-signature. Sure it would be better to have all parameters for every funtion explained in the parameter, but I don't think I can handle this right now so I just searched for "wrong" references, not missing ones.